src/http/routing.rs

```diff
@@ -43,15 +43,15 @@ pub enum Route {
     /// The data is the relevant [`ChannelId`].
     ///
     /// [`ChannelId`]: crate::model::id::ChannelId
```

Comment:
Can you move this line above the `[Rate Limits]:` link?

Reply:
Fixed